src/components/FeatureInformation/FeatureInformation.js

```diff
@@ -18,9 +18,21 @@ const FeatureInformation = ({ clickedFeatureInfo }) => {
   const { t } = useTranslation();
   const dispatch = useDispatch();
   const [featureIndex, setFeatureIndex] = useState(0);
-  const features = clickedFeatureInfo.map(l => l.features).flat();
+  const clickedFeatures = clickedFeatureInfo.map(l => l.features).flat();
+
+  // Extract cluster
```

Comment:
@oterral These are changes in core. Let's discuss if you have a better idea to show cluster popups (`with f.get('features')`).

Reply:
I added a comment to the ticket. For me, the flattening of custom features should not happen in here.

Reply:
I suggest to flatten the features in `getFeatureInfoAtCoordinate()` of the `ConstructionLayer`, However, see question there...